index.html

```diff
@@ -3744,17 +3869,41 @@ <h2>
             further action. The <a>user agent</a> SHOULD ensure that this never
             occurs.
           </li>
-          <li>Let <var>response</var> be a new <a>PaymentResponse</a>.
+          <li>Let <var>isRetry</var> be true if
+          <var>request</var><a>[[\response]]</a> is not null, false otherwise.
           </li>
-          <li>Set the <a data-lt="PaymentResponse.requestId">requestId</a>
-          attribute value of <var>response</var> to the value of
-          <var>request</var>.<a>[[\details]]</a>.<a data-lt=
-          "PaymentDetailsInit.id">id</a>.
+          <li>Let <var>response</var> be <var>request</var><a>[[\response]]</a>
+          if <var>isRetry</var> is true, or a new <a>PaymentResponse</a>
+          otherwise.
           </li>
-          <li>Set the <a data-lt="PaymentResponse.methodName">methodName</a>
-          attribute value of <var>response</var> to the <a>payment method
-          identifier</a> for the <a>payment method</a> that the user selected
-          to accept the payment.
+          <li>If <var>isRetry</var> if false, initialize the newly created
+          <var>response</var>:
+            <ol>
+              <li>Set <var>response</var>.<a>[[\request]]</a> to
+              <var>request</var>.
+              </li>
+              <li>Set <var>response</var>.<a>[[\retrying]]</a> to false.
+              </li>
+              <li>Set <var>response</var>.<a>[[\retryPromise]]</a> to
+              undefined.
+              </li>
+              <li>Set <var>response</var>.<a>[[\complete]]</a> to false.
+              </li>
+              <li>Set the <a data-lt="PaymentResponse.requestId">requestId</a>
+              attribute value of <var>response</var> to the value of
+              <var>request</var>.<a>[[\details]]</a>.<a data-lt=
+              "PaymentDetailsInit.id">id</a>.
+              </li>
+              <li>Set the <a data-lt=
```

Comment:
@zkoch, question for you: upon a retry, will it be possible for a user to change payment handlers. For instance, can a user switch from "basic-card" to "https://bob.pay"? Seems like they probably should be able to.

Reply:
My view is yes, they should be able to do so.

Reply:
I'll move it down, so it's not part of the initialization.